crates/trie/sparse/src/trie.rs

```diff
@@ -129,34 +193,66 @@ impl<P> SparseTrie<P> {
         Ok(self.as_revealed_mut().unwrap())
     }
 
-    /// Wipe the trie, removing all values and nodes, and replacing the root with an empty node.
+    /// Wipes the trie by removing all nodes and values,
+    /// and resetting the trie to only contain an empty root node.
+    ///
+    /// This is useful for when you want to reuse a trie instance without
+    /// creating a new one, clearing all of its contents.
```

Comment:
This is not entirely correct, this method is used in case of SELFDESTRUCT when we want to clear the storage trie contents. Calling `wipe` will also track the wipe in the updates, so it will be reflected in `take_updates`.